capture/SharpCap/samples/Totality.py

```diff
@@ -125,5 +98,10 @@ def Image(exposure, between_exp):
                     Added reset values for Exposure/Resolution/Gain for returning to live mode
                     Moved run_name variable to User input section
                     
-                    
+                  
```

Comment:
These are better in your commit message.  That lets someone look over the revision history in github rather rthan digging into the code. 

The convention is one line summary / blank line / comments.